packages/layout/src/components/__tests__/insurer.spec.tsx

```diff
@@ -3,26 +3,28 @@ import { render } from '@testing-library/react';
 import { InsurerCard } from '../cards/insurer/insurer';
 import { axe } from 'jest-axe';
 import userEvent from '@testing-library/user-event';
+import { Insurer } from '../cards/insurer/context';
 
 // TODO: write more tests
 
 const noop = () => Promise.resolve();
 
-const insurer = {
+const insurer: Insurer = {
 	id: '',
-	schemeRoleId: 123,
+	schemeRoleId: '123',
 	effectiveDate: '1997-04-01T00:00:00',
 	organisationReference: 123,
 	organisationName: 'Some Organisation Name',
 	insurerCompanyReference: '12345678',
-	addressLine1: 'Napier House',
-	addressLine2: 'Trafalgar Pl',
-	addressLine3: '',
-	postTown: 'Brighton',
-	postCode: 'BN1 4DW',
-	county: 'West Sussex',
-	postcode: '',
```

Comment:
We're trying to standardise on postcode as one word rather than two throughout the app (ie postcode, not postCode).

Reply:
update: using "postcode" now everywhere